Project/Fall2020_CSC403_Project/TextEngine.cs

```diff
@@ -224,6 +271,35 @@ private void HandleMarkup(string line)
                 case Markup.CheckThresholdsForTree:
                     // line is: empty, just check thresholds
                     break;
+                case Markup.ExitOptions:
+                    //  HandleMarkup(Story.GetNextLine());
```

Comment:
It'd be good if you added a comment here saying what the line being read in looked like and what would happen

Reply:
gonna add comments :)